js/jquery/plugins/jquery.colorpicker.js

```diff
@@ -63,6 +59,16 @@
     return this;
   };
 
+  $.fn.mColorPicker.setTextColor = function(element) {
+    try {
+      element.css({
+        'background-color': element.val()
+      }).css({
+        'color': $.fn.mColorPicker.textColor(element.css('background-color'))
+      }).trigger('change');
+    } catch (r) {}
+    };
```

Comment:
```suggestion
  };
```
Oups adding unneeded spaces, my bad ^^